public/locales/en-US/send.ftl

```diff
@@ -25,6 +25,15 @@ uploadingFileNotification = Notify me when the upload is complete.
 uploadSuccessConfirmHeader = Ready to Send
 uploadSvgAlt = Upload
 uploadSuccessTimingHeader = The link to your file will expire after 1 download or in 24 hours.
+expireInfo = The link to your file will expire after { $downloadCount } or { $timespan }.
+downloadCount = { $number ->
+        [one] 1 download
+       *[other] { $number } downloads
+}
```

Comment:
You need to indent the last curly parenthesis, e.g.
http://projectfluent.io/fluent/guide/selectors.html
https://github.com/mozilla-services/screenshots/blob/master/locales/en-US/server.ftl#L161